project/sites/cvocni/themes/cvocni_theme/src/scss/components.scss

```diff
@@ -114,6 +114,10 @@
   }
 }
 
+.sidebar-second .content-sidebar-wrap {
+  width: 65%;
```

Comment:
There is a variable that Orla created to work this out, so no need to add it like this. In your _variables.scss you can add this for second-sidebar $two-col-layout-sidebar-second: (65%, 30%);
Or if you have a first-sidebar $two-col-layout-sidebar-first: (30%, 65%);

Reply:
Done, thank you